packages/paste-website/src/pages/patterns/status/index.mdx

```diff
@@ -183,6 +212,52 @@ used to 3 or 4 of the most important statuses. For statuses that are not as crit
   {processStatusExamples}
 </LivePreview>
 
+To determine how to show a process status:
+- If additional information is needed about the status, use Icon, Text, and [Anchor](/components/anchor) (optional).
+- If it’s in a Table and no additional information is needed, use a [borderless Status Badge](/components/status-badge#borderless-badges).
+- Otherwise, use the default [Status Badge](/components/status-badge).
+Placement of the status in a Table should depend on the importance of the status to the given user flow and should be tested with users.
+For statuses that are not as critical to a user’s workflow, use [plain text statuses](#plain-text-status).
```

Comment:
```suggestion
- Otherwise, use the default [Status Badge](/components/status-badge).

Placement of the status in a Table should depend on the importance of the status to the given user flow and should be tested with users.

For statuses that are not as critical to a user’s workflow, use [plain text statuses](#plain-text-status).
```